source/guides/private-bitbucket-repository.rst

```diff
@@ -18,6 +18,7 @@ For this tutorial you will need:
 * a private Bitbucket repository
 * a Valohai project which to link the repository
 * a tool that generates SSH keys, this guide uses :code:`ssh-keygen`
+* access from Valohai (IP address: 34.248.245.191) through your firewall to your private Bitbucket
```

Comment:
@akx do you think there is any reason to keep this hidden? I guess this could make it possible to do some dossing